composer.json

```
@@ -28,7 +28,7 @@
         "ext-json": "*"
     },
     "require-dev": {
-        "sminnee/phpunit": "^5",
+        "phpunit/phpunit": "^9",
```

```suggestion
        "phpunit/phpunit": "^9.5",
```

Maybe add a PHP constraint as well.

---

Update